### DISCUSSION TITLE
Page to next column

### DISCUSSION THREAD

**Discussion by Kazagan** [QuestPDF/QuestPDF#755] [Q&A] [answered]
If i were to split my document into 2 columns, using the row feature of quest pdf, is there a way i could tell questpdf to page into the next column before paging into the next page, and returning back to the first column.

Im on phone right now so formating here migh lt be bad:
| A.     |  B.     |
| A.     |  B.     |
| A.     |  B.     |
| A.     |  B.     |

| C.     |  D.     |
| C.     |  D.     |
| C.     |  D.     |
| C.     |  D.     |

So in the above, i have 2 pages, if i want to render a table element starting in column A, if it were to be to long, and need to page, can i make it go to column B before going to column C and then into D.

**Comment by MarcinZiabek** [comment]
I am sorry to say that this particular feature is not available now. However, I have a very promising prototype that I want to explore soon. Unfortunately, this requirement introduces non-trivial changes to the layout algorithm (https://github.com/QuestPDF/QuestPDF/tree/feat-multicolumn-prototype)

That being said, you can utilize existing features if you have additional knowledge about your content. For example, if you want to render a list of items and do not require those items to split/page, you can use the `.RotateRight().Inlined(x => x.Item().RotateLeft()...)` construct.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

The functionality you described is now available through the [MultiColumn](https://www.questpdf.com/api-reference/multi-column.html) element, introduced in version 2024.7.0.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
